### PR TITLE
Fixed #8873 - Cancellation emails are being sent to both the customer…

### LINKED ISSUE #8873: The order cancellation emails are not being sent correctly. The emails are not being sent to the intended recipients.
### Terms

- [X] Before you openning this issue, i have checked if the issue has already been reported.

### Bagisto Version(s) affected

master

### Issue Description

When an order is canceled, an email is sent to the customer, but the email is not being sent to the admin and the email which is currently being sent to the customer should be sent to the administrator and another mail to the customer.

### Preconditions

1. Admin should be configured mail.
2. Admin should have at least one product created.

### Steps to reproduce

1. Go to 'Shop'
2. Add a product to the cart and proceed to checkout.
3. Fill in all the required fields and select the preferred shipping/payment method.
4. Click on the "Place Order" button.
5. Access the 'Admin Panel.'
6. Open the order and cancel it.
7. Check the mailbox of the customer and the admin.
8. See the error.

### Expected Result

The email which is currently being sent to the customer should be sent to the administrator and another mail to the customer.

### Actual Result

When an order is canceled, an email is sent to the customer, but the email is not being sent to the admin and the email which is currently being sent to the customer should be sent to the administrator and another mail to the customer.

### Screenshots

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by sunarcrajneesh**
… and the admin correctly.

## Issue Reference

#8873 

## Description

Cancellation emails are being sent to both the customer and the admin correctly.

## How To Test This?

- Go to 'Shop'
- Add a product to the cart and proceed to checkout.
- Fill in all the required fields and select the preferred shipping/payment method.
- Click on the "Place Order" button.
- Access the 'Admin Panel.'
- Open the order and cancel it.
- Check the mailbox of the customer and the admin.

## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

**Comment by Amitk-webkul** [comment]
@sunarcrajneesh It is working fine.

File: packages/Webkul/Admin/src/Mail/Order/CanceledNotification.php [hunk 1]
```diff
@@ -21,7 +21,7 @@ public function __construct(public $order)
     public function build()
     {
         return $this->from(core()->getSenderEmailDetails()['email'], core()->getSenderEmailDetails()['name'])
-            ->to($this->order->customer_email, $this->order->customer_full_name)
+            ->to(core()->getAdminEmailDetails()['email'], core()->getAdminEmailDetails()['name'])
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Mail/Order/CanceledNotification.php:24]: No Need to changes here.

File: packages/Webkul/Admin/src/Resources/views/emails/orders/canceled.blade.php [hunk 2]
```diff
@@ -5,7 +5,7 @@
         </span> <br>
 
         <p style="font-size: 16px;color: #5E5E5E;line-height: 24px;">
-            @lang('admin::app.emails.dear', ['customer_name' => $order->customer_full_name]),👋
+            @lang('admin::app.emails.dear', ['admin_name' => core()->getAdminEmailDetails()['name']]),👋
```
Comment (amit-webkul) [inline packages/Webkul/Admin/src/Resources/views/emails/orders/canceled.blade.php:8]: We have Fix this issue.
Please revert the changes in the commented section.

File: packages/Webkul/Admin/src/Mail/Order/CanceledNotification.php (hunk 1 above)
Comment (sunarcrajneesh) [reply packages/Webkul/Admin/src/Mail/Order/CanceledNotification.php:24]: @amit-webkul,
If we don't change here, the cancellation email will be sent to the customer twice, and the admin will not receive it.

Comment (amit-webkul) [reply packages/Webkul/Admin/src/Mail/Order/CanceledNotification.php:24]: Here, mail is delivered to a single customer. But when we look from an admin perspective, we need to add these changes.

**Thank you for this discussion.**

**Review by amit-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**